middlewares/auth.middleware.js

```diff
@@ -12,9 +13,10 @@ const isAdmin = (req, res, next) => {
   if (req.isAuthenticated() && req.user.isAdmin) {
     next();
   } else {
+    console.log(req.user);
     res
       .status(401)
-      .json({ msg: "You are not authorized to view this resource." });
+      .json({ msg: "You are not admin" });
```

Comment:
Can we update this wording to this? 
"You do not have admin permissions."

Reply:
Thank you for the feedback